config/go-app-ussd_popi_user_data.afr_ZA.json

```diff
@@ -209,19 +209,19 @@
     ],
     "Invalid phone number. Please re-enter (with no spaces)": [
         null,
-        ""
+        "Ongeldige selnommer. Tik asb weer in"
     ],
     "Sorry, the number you are trying to move to already has an active registration. To manage that registration, please redial from that number.": [
         null,
-        ""
+        "Jammer, die nommer wat jy nou wil gebruik het reeds 'n aktiewe registrasie. Om daardie registrasie te bestuur moet jy weer daardie nommer bel"
     ],
     "Thank you. Your info has been updated.": [
         null,
         "Baie dankie. Jou info is bygewerk."
     ],
     "Your personal information stored on MomConnect has not been removed.": [
         null,
-        ""
+        "Jou persoonlike info wat op MomConnect gestoor is, is verwyder"
```

Comment:
This means the exact opposite of the english 🙂 
"Jou persoonlike info wat op MomConnect gestoor is, is nie verwyder" - maybe